_data/companies/smalan-norge.yml

```diff
@@ -0,0 +1,6 @@
+name: Smalan-norge [[1]](https://smalan-norge.com/smalan-norge-coronavirus/) 
```

Comment:
Hi,

That link gives me a (very adorable) 404.

![image](https://user-images.githubusercontent.com/7773256/77422581-b7d88400-6d9b-11ea-9ce6-9f5a9b618b93.png)

Reply:
Hello. Please check again. Should be good

Reply:
![image](https://user-images.githubusercontent.com/7773256/77555138-e9794a00-6e84-11ea-9f61-ef58e9e49d40.png)

Weirdly, I still see it.